### DISCUSSION TITLE
Data correctness issues

### DISCUSSION THREAD

**Discussion by bill-e-ghote** [projectdiscovery/asnmap#6] [General]
I queried asnmap for an IP / ASN known to me through work. See attached screenshot.

![image](https://user-images.githubusercontent.com/2491041/193975467-78ecb35b-d013-48b4-a9ee-f7d06c5b1bb4.png)

For comparison purposes, here's a screenshot of the same query against Team Cymru's IP to ASN lookup.

![image](https://user-images.githubusercontent.com/2491041/193976843-a27cc1a4-8ea7-44a2-acf6-8d49af3b2b9d.png)

The Cymru mapping to NYLI-AS is correct. The asnmap data is not, or at least does not include the distinct mapping to NYLI-AS. 

Even the Cymru data is not complete. The network allocation is larger than /24, it's /19. https://whois.domaintools.com/206.210.19.0

**Comment by ehsandeep** [comment]
Thank you @bill-e-ghote for pointing this out, unfortunately we can not do much about it at this moment as we are relying on [iptoasn](https://iptoasn.com) db for this information.

**Reply by bill-e-ghote** [reply]
Ack. Thanks for the feedback.
